tests/conftest.py

```diff
@@ -118,37 +118,33 @@ def firefox_options(
     """Setup Firefox"""
     firefox_options.log.level = "trace"
-    # if request.config.getoption("--run-update-test"):
-    #     # if request.node.get_closest_marker(
-    #     #     "update_test"
-    #     # ):  # disable test needs different firefox
-    #     #     binary = Path(
-    #     #         "utilities/firefox-old-nightly-disable-test/firefox/firefox-bin"
-    #     #     ).absolute()
-    #     #     firefox_options.binary = f"{binary}"
-    #     #     firefox_options.add_argument("-profile")
-    #     #     firefox_options.add_argument(
-    #     #         f'{Path("utilities/klaatu-profile-disable-test").absolute()}'
-    #     #     )
-    #     # else:
-    #     binary = Path(
-    #         "utilities/firefox-old-nightly/firefox/firefox-bin"
-    #     ).absolute()
-    #     firefox_options.binary = f"{binary}"
-    #     firefox_options.add_argument("-profile")
-    #     firefox_options.add_argument(setup_profile)
-    # if request.config.getoption("--run-firefox-release"):
-    #     binary = Path("utilities/firefox-release/firefox/firefox-bin").absolute()
-    #     firefox_options.binary = f"{binary}"
-    # if request.node.get_closest_marker("reuse_profile") and not request.config.getoption(
-    #     "--run-update-test"
-    # ):
-    #     firefox_options.add_argument("-profile")
-    #     firefox_options.add_argument(setup_profile)
+    if request.config.getoption("--run-update-test"):
+        if request.node.get_closest_marker("update_test"):  # disable test needs different firefox
+            binary = Path(
+                "utilities/firefox-old-nightly-disable-test/firefox/firefox-bin"
+            ).absolute()
+            firefox_options.binary = f"{binary}"
+            firefox_options.add_argument("-profile")
+            firefox_options.add_argument(
+                f'{Path("utilities/klaatu-profile-disable-test").absolute()}'
+            )
+        else:
+            binary = Path("utilities/firefox-old-nightly/firefox/firefox-bin").absolute()
+        firefox_options.binary = f"{binary}"
+        firefox_options.add_argument("-profile")
+        firefox_options.add_argument(setup_profile)
+    if request.config.getoption("--run-firefox-release"):
+        binary = Path("utilities/firefox-release/firefox/firefox-bin").absolute()
+        firefox_options.binary = f"{binary}"
+    if request.node.get_closest_marker("reuse_profile") and not request.config.getoption(
+        "--run-update-test"
+    ):
+        firefox_options.add_argument("-profile")
+        firefox_options.add_argument(setup_profile)
     firefox_options.set_preference("extensions.install.requireBuiltInCerts", False)
     firefox_options.log.level = "trace"
     firefox_options.set_preference("browser.cache.disk.smart_size.enabled", False)
-    firefox_options.set_preference("toolkit.telemetry.server", "http://localhost:5000")
+    firefox_options.set_preference("toolkit.telemetry.server", "http://ping-server:5000")
```

Comment:
is this always 5000? I thought we were using a `os.environ.get("PORT", 5000)` above so curious if that could get overridden and out of sync maybe.

https://github.com/mozilla/klaatu/blob/a3102f0c3a962e8f665bec06f468082379832cd2/ping_server/ping_server.py#L48

Reply:
Yeah this is always ```:5000```